openapi/operation.go

```diff
@@ -29,11 +30,17 @@ func (o *Operation) getChildren() map[string]Traversable {
 	if o == nil {
 		return children
 	}
+	// Parameters
 	for i := range o.Parameters {
 		parameter := o.Parameters[i]
-		children[string(i)] = parameter
+		children[string(rune(i))] = &parameter
```

Comment:
I don't think this is doing what you want it to do:
https://go.dev/play/p/Ti1cl_OQuTF

Reply:
at the time I was just taking intellij's recommendation :|

Reply:
opted for `strconv.Itoa()` instead